code/modules/admin/admin.dm

```diff
@@ -782,6 +782,22 @@ var/global/floorIsLava = 0
 	SSstatistics.add_field_details("admin_verb","TLOOC") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
 
+/datum/admins/proc/endnow()
+	set category = "Server"
+	set desc = "round goes bye bye"
```

Comment:
No concerns about this, although other people might. We perform an infinitesimal amount of mischief.